packages/core/src/index.ts

```diff
@@ -2,7 +2,8 @@ export {
   executeRequest,
   BaseLlmParametersWithDeploymentId,
   BaseLlmParameters,
-  CustomRequestConfig
+  CustomRequestConfig,
+  EndpointOptions
```

Comment:
[q] Is it intended to make this part of the public API? Looks somewhat "internal" to me.

Reply:
Yeah I had a brief discussion about it and it is intended to be public.